docs/usage/validation.md

```diff
@@ -25,7 +25,11 @@ render((
 
 ## HTML5 Validation
 
-By default, the form uses HTML5 validation. This may cause unintuitive results because the HTML5 validation errors (such as when a field is `required`) may be displayed before the form is submitted, and thus these errors will display differently from the react-jsonschema-form validation errors. You can turn off HTML validation by setting the `noHtml5Validate` to `true`.
+By default, the form does not use HTML5 validation. 
+
+If you wish to enable HTML5 validation, passing `noHtml5Validate=false` will enable it. 
```

Comment:
Do you think this is intuitive enough (seems kind of strange that someone would need to pass in "false" to enable a feature)? Would it make better sense to create a new prop called `html5Validate=true`, or do you think `noHtml5Validate=false` is clear enough?

Reply:
I was mostly keeping the props consistent, but if you're cool with swapping that out, I think that makes more sense.   I'll udpate the code/tests/docs to do so

Reply:
Yeah, I guess I'm not sure. I'm cool with swapping it out if it makes sense, do you think it makes sense to do so?